packages/@webex/plugin-meetings/src/interceptors/locusRetry.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@webex/http-core';
+
+const rateLimitExpiryTime = new WeakMap();
+/**
+ * @class
+ */
+export default class LocusRetryStatusInterceptor extends Interceptor {
```

Comment:
I dont understand why this is a Locus specific retry status interceptor. There's nothing in here specific to Locus. I know it seems like it's just a naming thing but this retry interceptor should be usable for other services that need to be retried based on unavailability.

Also is there any reason why this retry interceptor doesn't handle the x-retry-after-header for 429s as well as 503 ? why can't this be a generic http retry interceptor that could be extended?

Reply:
This is incorrect. This is specifically a single retry for locus. This does not perform any backoff logic. Other services try a different regional cluster when a failure of this scope occurs (it marks the host node as failed, see the services plugin catalog [ `packages/@webex/webex-core/src/lib/services/service-catalog.js` ]. ) This is a server-side failover workflow specific for locus.